src/Veldrid.OpenGLBindings/OpenGLNative.cs

```diff
@@ -1884,6 +1885,12 @@ public static void LoadAllFunctions(IntPtr glContext, Func<string, IntPtr> getPr
                     LoadFunction("glCopyImageSubDataEXT", out p_glCopyImageSubData);
                 }
 
+                LoadFunction("glTextureView", out p_glTextureView);
+                if(p_glTextureView == null)
+                {
+                    LoadFunction("glTextureViewOES", out p_glTextureView);
```

Comment:
I don't have a clean test for this.  The phone that I have that supports this extension ends up with the deadlocking issue described [here](https://community.khronos.org/t/adreno-510-gpu-deadlock-would-occur-when-trying-to-use-ubo-in-fragment-shader/106773/3).